parts/gameFuncs.lua

```diff
@@ -1010,6 +1045,7 @@ do-- function resetGameData(args)
                 TEXT.show(M.id,640+(counter%5-2)*80,26,45,'spin')
                 counter=counter+1
             end
+            time=time+1
```

Comment:
There is a small bug here
![image](https://github.com/26F-Studio/Techmino/assets/106439598/9ad46b50-f85e-4995-983f-b5d0cde3de99)

You may want to change like this below, but don't if it is not the thing you want
```suggestion
time=time and time+1 or 0
```
But change like this may lead to
![image](https://github.com/26F-Studio/Techmino/assets/106439598/74a4fd6a-30d2-4b62-8b1a-35d858e99678)
I don't know if it is a thing you expected to see or not